app/build.gradle

```diff
@@ -134,5 +134,8 @@ dependencies {
     androidTestImplementation "com.android.support.test.espresso:espresso-core:$rootProject.espressoVersion"
     androidTestImplementation "com.android.support.test:runner:$rootProject.runnerVersion"
     androidTestImplementation "com.android.support.test:rules:$rootProject.rulesVersion"
+
+    //Multidex
```

Comment:
@ethan627hsu Create a separate PR for this as its not relevant for this issue and undo these changes.